### PR TITLE
Did overall formatting of the code as per java standards

### PR DESCRIPTION
**Pull request by Rajesh734**

### CODE REVIEW

File: spring-boot-mysql-jpa-hibernate-swagger-master_spring-boot-mysql-jpa-hibernate-swagger-master/src/main/java/com/springboot/Application.java [hunk 1]
```diff
@@ -8,8 +8,8 @@
 @EnableTransactionManagement
 public class Application {
 
-  public static void main(String[] args) {
-    SpringApplication.run(Application.class, args);
-  }
+	public static void main(String[] args) {
```
Comment (Selsdeepika) [inline spring-boot-mysql-jpa-hibernate-swagger-master_spring-boot-mysql-jpa-hibernate-swagger-master/src/main/java/com/springboot/Application.java]: should not use Tab for formatting

**Review by Selsdeepika — CHANGES_REQUESTED** [review]
Please work on my review comment

File: spring-boot-mysql-jpa-hibernate-swagger-master_spring-boot-mysql-jpa-hibernate-swagger-master/src/main/java/com/springboot/Application.java (hunk 1 above)
Comment (Rajesh734) [reply spring-boot-mysql-jpa-hibernate-swagger-master_spring-boot-mysql-jpa-hibernate-swagger-master/src/main/java/com/springboot/Application.java]: Done

Comment (Selsdeepika) [reply spring-boot-mysql-jpa-hibernate-swagger-master_spring-boot-mysql-jpa-hibernate-swagger-master/src/main/java/com/springboot/Application.java]: Ok updated to the source

**Pull request merged**